### PR TITLE
Pythonapi coco

### PR DESCRIPTION
**Pull request by bellycat77**
Pycocotools (Python API for COCO-dataset) are added

### CODE REVIEW

File: package/pythonapi-coco/install.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# Standard env.
+# INSTALL_DIR
+# PACKAGE_DIR
+
+# Custom env (./cm/meta.json).
+# PACKAGE_URL
+# PACKAGE_BRANCH
+# PACKAGE_NAME
+# LIB_NAME
```
Comment (psyhtest) [inline package/pythonapi-coco/install.sh]: Unused?

File: package/pythonapi-coco/.cm/meta.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "customize": {
+    "extra_dir": "", 
+    "install_env": {
+      "PACKAGE_BRANCH": "master", 
+      "PACKAGE_NAME": "pythonapi-coco", 
+      "PACKAGE_URL": "https://github.com/cocodataset/cocoapi.git", 
+      "TOOL_NAME": "pythonapi-coco"
```
Comment (psyhtest) [inline package/pythonapi-coco/.cm/meta.json]: I'd recomment to use `tool-cocoapi` or simply `tool-coco`. We don't use `pythonapi-*` anywhere else.

File: package/pythonapi-coco/.cm/meta.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  }, 
+  "only_for_host_os_tags": [
+    "linux"
+  ], 
+  "only_for_target_os_tags": [
+    "linux"
+  ], 
+  "process_script": "install", 
+  "soft_uoa": "7432c898273fe70d", 
+  "suggested_path": "pythonapi-coco",
```
Comment (psyhtest) [inline package/pythonapi-coco/.cm/meta.json]: tool-coco

File: soft/pythonapi.coco/.cm/info.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "engine": "CK", 
+    "iso_datetime": "2018-10-05T16:44:42.931537", 
+    "license": "See CK LICENSE.txt for licensing details", 
+    "version": [
+      "1", 
+      "9", 
+      "5"
+    ]
+  }, 
+  "data_name": "pythonapi.coco"
```
Comment (psyhtest) [inline soft/pythonapi.coco/.cm/info.json]: `tool.coco`?

File: soft/pythonapi.coco/.cm/meta.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "auto_detect": "yes",
+  "customize": {
+    "env_prefix": "CK_ENV_PYTHONAPI_COCO",
```
Comment (psyhtest) [inline soft/pythonapi.coco/.cm/meta.json]: `CK_ENV_TOOL_COCO`?

File: soft/pythonapi.coco/README.md [hunk 6]
```diff
@@ -0,0 +1 @@
+COCO dataset Python PAI
```
[the remark this answers is not in the document]
Comment (bellycat77) [reply soft/pythonapi.coco/README.md]: Done

File: soft/pythonapi.coco/.cm/meta.json (hunk 5 above)
Comment (bellycat77) [reply soft/pythonapi.coco/.cm/meta.json]: Done

**Pull request merged**